src/tests/Footer.test.js

```diff
@@ -0,0 +1,26 @@
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import Footer from '../components/Footer';
+
+describe('Test the funtionalities of footer', () => {
+  const footerEl = render(<Footer />);
+
+  // Create snapshot for later matching
+  test('Match the snapshot', () => {
+    expect(footerEl).toMatchSnapshot();
+  });
```

Comment:
- [ ] As mentioned in the project requirements all the tests should pass, please kindly make sure all your tests are passing 🙃 😉 

<img width="1068" alt="Screenshot 2023-07-12 at 23 58 18" src="https://github.com/M-Anwar-Hussaini/Math-Magicians/assets/74501523/cabc1cb1-1a39-4941-a6b8-827e13bc816f">

Reply:
Hi @cynthiainga .
Thanks a lot for reviewing the code. I guess the reason why this test is failing is, because of the snapshot that is taken which shows a live time displaying on the page. I guess for footer snapshot is not needed.